services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/PathNodePayload.java

```diff
@@ -41,19 +42,21 @@
      * Input port number.
      */
     @JsonProperty("input_port")
-    private int inputPort;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Integer inputPort;
 
     /**
      * Output port number.
      */
     @JsonProperty("output_port")
-    private int outputPort;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Integer outputPort;
```

Comment:
what's the reason for Object instead of primitive here?

Reply:
I need it to use `null` if we this is no information about ports and do not include it into API response. In my case it happens for first (no input_port) and last (no output_port) switch